analyzer/runtime/runtime.go

```diff
@@ -286,10 +286,7 @@ func (m *processor) queueDbUpdates(batch *storage.QueryBatch, data *BlockData) {
 
 	// Insert EVM token addresses.
 	for addr, possibleToken := range data.PossibleTokens {
-		totalSupply := "0"
-		if possibleToken.TotalSupplyChange != nil {
-			totalSupply = possibleToken.TotalSupplyChange.String()
-		}
+		totalSupply := possibleToken.TotalSupplyChange.String()
```

Comment:
`totalSupplyChange` please, or `totalSupplyChangeStr`. Or pretty much anything that has `change` or `delta` in the name :)